channeldb/channel.go

```diff
@@ -3216,6 +3267,441 @@ const (
 	Abandoned ClosureType = 5
 )
 
+// LocalForceCloseInsights holds information about a local force close.
+type LocalForceCloseInsights struct {
+	// Initiator indicates what initiated the local force close.
+	Initiator LocalForceCloseInitiator
```

Comment:
Why is it a "Local" ForceCloseInitiator?

Reply:
Because it is only for local force close. Follow up PR could be for remote force close.